php/src/Communicator.cpp

```diff
@@ -165,8 +165,9 @@ class ValueFactoryManager : public Ice::ValueFactoryManager
 {
 public:
 
+    virtual void add(Ice::ValueFactoryFunc, const string&);
     virtual void add(const Ice::ValueFactoryPtr&, const string&);
-    virtual Ice::ValueFactoryPtr find(const string&) const noexcept;
+    virtual Ice::ValueFactoryFunc find(const string&) const noexcept;
```

Comment:
I think these should be baked override too.

Reply:
Fixed. I used 'final'.